deploy-test-apps and deploy-full-local-setup make commands are broken

First user experience commands (`make deploy-test-apps` and `make deploy-full-local-setup`) are broken after #738 merge:

  ```sh
      $ make deploy-full-local-setup
      ...
      Deploy test apps
      kubectl config use-context k3d- test-gslb1
      Set the current-context in a kubeconfig file.
  
      Aliases:
      use-context, use
  
      Examples:
      # Use the context for the minikube cluster
      kubectl config use-context minikube
  
      Usage:
      kubectl config use-context CONTEXT_NAME [options]
  
      Use "kubectl options" for a list of global command-line options (applies to all commands).
      error: Unexpected args: [k3d- test-gslb1]
      make: *** [deploy-full-local-setup] Error 1
  ```

  ```sh
      $ make deploy-test-apps
      ...
      Deploy test apps
      kubectl config use-context k3d-
      Set the current-context in a kubeconfig file.
  
      Aliases:
      use-context, use
  
      Examples:
      # Use the context for the minikube cluster
      kubectl config use-context minikube
  
      Usage:
      kubectl config use-context CONTEXT_NAME [options]
  
      Use "kubectl options" for a list of global command-line options (applies to all commands).
      error: Unexpected args: [k3d- test-gslb1]
      make: *** [deploy-test-apps] Error 1
  ```

--- a/Makefile
+++ b/Makefile
@@ -120,9 +120,8 @@ deploy-full-local-setup: ## Deploy full local multicluster setup (k3d >= 4.2.0)
 	$(call deploy-local-cluster,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),$(VERSION),$(CLUSTER_GSLB2_HELM_ARGS),'k8gb/k8gb')
 
 	@echo "\n$(YELLOW)Deploy test apps $(NC)"
-	$(call deploy-test-apps, $(CLUSTER_GSLB1))
-	$(call deploy-test-apps, $(CLUSTER_GSLB2))
-
+	$(call deploy-test-apps,$(CLUSTER_GSLB1))
+	$(call deploy-test-apps,$(CLUSTER_GSLB2))
 
 .PHONY: deploy-stable-version
 deploy-stable-version:
@@ -443,12 +442,15 @@ define apply-cr
 endef
 
 define deploy-test-apps
-	kubectl config use-context k3d-$1
-	@echo "\n$(YELLOW)Deploy GSLB cr $(NC)"
+	$(if $1,@kubectl config use-context k3d-$1, \
+		@echo "Current context: $(shell kubectl config current-context)")
+
+	@echo "\n$(YELLOW)Deploy GSLB CR $(NC)"
 	kubectl apply -f deploy/crds/test-namespace.yaml
 	$(call apply-cr,deploy/crds/k8gb.absa.oss_v1beta1_gslb_cr.yaml)
 	$(call apply-cr,deploy/crds/k8gb.absa.oss_v1beta1_gslb_cr_failover.yaml)
 
+	@echo "\n$(YELLOW)Deploy podinfo $(NC)"
 	kubectl apply -f deploy/test-apps
 	helm repo add podinfo https://stefanprodan.github.io/podinfo
 	helm upgrade --install frontend --namespace test-gslb -f deploy/test-apps/podinfo/podinfo-values.yaml \

